src/client/eppo-client.ts

```diff
@@ -1066,8 +1427,19 @@ export default class EppoClient {
       );
     }
 
-    const configDetails = config.getFlagConfigDetails();
-    const flag = this.getNormalizedFlag(config, flagKey);
+    const precomputed = config.getPrecomputedConfiguration();
+    if (precomputed && precomputed.subjectKey === subjectKey) {
+      // Short-circuit evaluation if we have a matching precomputed configuration.
+      const precomputedResult = this.evaluatePrecomputedAssignment(
+        precomputed,
+        flagKey,
+        expectedVariationType,
+      );
+
+      return precomputedResult.flagEvaluation;
+    }
+
+    const flag = config.getFlag(flagKey);
```

Comment:
EppoClient now automatically handles precomputed assignments if precomputed configuration is present for the specified subject key